cockroachcloud/migrations-page.md

```diff
@@ -139,64 +139,88 @@ If the credentials are valid, they will be added to the [**Credentials** table](
 
 ## Review the schema
 
+Use the **Summary Report** and **Statements** list to [update the schema](#update-the-schema) and finalize it for migration.
+
 ### Summary Report
 
-The **Summary Report** displays the results of the schema analysis:
+The **Summary Report** displays the results of the schema analysis and provides bulk actions you can apply to [update the schema](#update-the-schema).
+
+The banner at the top of the page displays:
 
 <ul>
 <li>The number of <b>Statements Total</b> in the uploaded <code>.sql</code> file that were analyzed.</li>
-<li>The number of <b>Errors</b> in SQL statements that are blocking <a href="#finalize-the-schema">finalization</a>. Errors are further categorized and counted on the <a href="#statement-status"><b>Statement Status</b></a> graph.</li>
+<li>The number of <b>Errors</b> in SQL statements that are blocking <a href="#migrate-the-schema">schema migration</a>. Errors are further categorized and counted on the <a href="#statement-status"><b>Statement Status</b></a> graph.</li>
 <li>The number of <b>Incidental Errors</b> in SQL statements that are caused by errors in other SQL statements.</li>
 
 <section class="filter-content" markdown="1" data-scope="mysql oracle mssql">
 <li>The number of <b>Incompatible Statements</b> that could not be converted because they have no equivalent syntax on CockroachDB.</li>
-<li>The number of <b>Compatibility Notes</b> regarding differences in SQL syntax. Although these statements do not block finalization, you should [update](#update-the-schema) them before finalization.</li>
+<li>The number of <b>Compatibility Notes</b> regarding differences in SQL syntax. Although these statements do not block schema migration, you should [update](#update-the-schema) them before migrating the schema.</li>
```

Comment:
Not sure if I'm misreading, but should it be "The number of compatibility notes pertain to differences in SQL syntax"?